embedded-workshop-book/src/beginner-next-steps.md

```diff
@@ -0,0 +1,36 @@
+# Next Steps
+
+## Collision avoidance
+
+Here's one more RADIO exercise for you: test out collision avoidance.
+
+If you check the API documentation of the `Radio` abstraction we have been using you'll notice that we haven't used these methods: `energy_detection_scan`, `set_cca` and `try_send`.
+
+The first method scans the currently selected channel (see `set_channel`), measures the energy level of ongoing radio communication in this channel and returns the maximum energy observed over a span of time. This method can be used to determine what the *idle* energy level of a channel is. If there's non-802.15.4 traffic ongoing on this channel the method will return a high value.
```

Comment:
```suggestion
The first method scans the currently selected channel (see `set_channe()l`), measures the energy level of ongoing radio communication in this channel and returns the maximum energy observed over a span of time. This method can be used to determine what the *idle* energy level of a channel is. If there's non-IEEE 802.15.4 traffic on this channel the method will return a high value.
```

Reply:
Maybe add a hint that this could for example be your wifi? people can for example use the graphic in https://inet.omnetpp.org/docs/showcases/wireless/coexistence/doc/ to check for overlaps between their wifi channel and the 802.15.4 channel used by dk+dongle

Reply:
ah, I just saw that you're explaining this later on. Would maybe still be helpful to add a graphic for mapping

Reply:
That's great info. I'll add the link.